tests/actions/test_core_actions.py

```diff
@@ -772,6 +775,18 @@ def test_action_deserialization(
         assert isinstance(actual, action_class)
 
 
+@contextmanager
+def mock_not_imported_module(
+    parent_module: ModuleType, target_name: str, mocker: MockerFixture
+) -> Iterator[Never]:
+    original = getattr(parent_module, target_name)
+    try:
+        setattr(parent_module, target_name, mocker.Mock())
+        yield getattr(parent_module, target_name)
```

Comment:
I originally had a bare `yield` which mypy didn't like and wanted to return a `NoReturn`. I tried that but it didn't immediately work and I didn't want to debug at the moment. I think this is fine since it returns the mocked value. Or we could look more at the type issue.